multiply.py

```diff
@@ -0,0 +1,7 @@
+list1 = [1, 2, 3]
+list2 = [4, 5, 6]
+
+result1 = numpy.prod(list1)
+result2 = numpy.prod(list2)
+print(result1)
```

Comment:
looks good, lets put it into a function.
 `def multiply_nums(list_of_nums):
  ...
` 
Also at the top of the file you need to add a line `import numpy`